kong/globalpatches.lua

```diff
@@ -101,6 +101,14 @@ return function(options)
       return timerng:every(interval, callback, ...)
     end
 
+    _G.named_timer_at = function (name, delay, callback, ...)
```

Comment:
i was thinking if we should properly move those functions that invokes timerng in a single table, like `kong.timer.named_at`.